c/tskit/tables.c

```diff
@@ -578,14 +584,46 @@ tsk_individual_table_get_row(
     return ret;
 }
 
+int
+tsk_individual_table_set_metadata_schema(tsk_individual_table_t *self,
+    const char *metadata_schema, tsk_size_t metadata_schema_length)
+{
+    int ret = 0;
+    tsk_safe_free(self->metadata_schema);
+    self->metadata_schema = NULL;
+    self->metadata_schema_length = metadata_schema_length;
+    if (metadata_schema != NULL) {
```

Comment:
Probably better to test if ``metadata_schema_length > 0`` here. If someone passes NULL with length > 0 it's probably a programming mistake and better to segfault below (or, we could test for this and error; but it's hard to know where to draw the line then with NULL inputs).

Reply:
Fixed in https://github.com/tskit-dev/tskit/pull/493/commits/01d615852dba1b64d5a463fccb869bf6338088fa